doc/tutorials/klondike/app/lib/main.dart

```diff
@@ -1,11 +1,12 @@
-import 'dart:html'; // ignore: avoid_web_libraries_in_flutter
+// import 'dart:html'; // ignore: avoid_web_libraries_in_flutter
```

Comment:
This should not be commented out, since this tutorial runs in the browser.

Reply:
This is just a hack I used to get this `main.dart` to run on Apple MacOS without compilation errors and a crash. We need to eradicate commit https://github.com/flame-engine/flame/commit/18f71bece67fad79b71b9185dc18101f81f0df27 which unintentionally added and committed this file.

Reply:
The offending commit https://github.com/flame-engine/flame/commit/18f71bece67fad79b71b9185dc18101f81f0df27 has been reverted.